### DISCUSSION TITLE
How to change drawer width?

### DISCUSSION THREAD

**Discussion by holmesjason** [zauberzeug/nicegui#161] [Q&A] [answered]
I can't figure this out.  Quasar specifies `:width="300"` as an example of how to set the width.  I have tried `ui.right_drawer(top_corner=True, bottom_corner=True).style(replace='background-color: #d7e3f4; width: 500')` and many combinations of "width", for example `:width: 500`, `:width: "500"`, and `width: 500`.  Does anyone know how to do this?

**Comment by falkoschindler** [comment, marked as answer]
These Quasar "props" need to be passed to the `.props()` method. The following example shows a very narrow drawer:

```python
@ui.page('/')
def page():
    ui.label('Content')
    with ui.left_drawer().props('width=80 bordered'):
        ui.label('Drawer')
```

In contrast, the `.style()` method expects CSS. But this has no effect in this case because Quasar does quite some magic in creating the page layout, so you can't manipulate the element's style directly.
